packages/camera/camera_android/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.9.7+1
+
+* Splits from `camera` as a federated implementation.
```

Comment:
do we want to add a link to the previous logs?

Reply:
People only use these packages via the app-facing package, so they'll already know where that changelog is. I wouldn't expect most people to ever see anything but the app-facing package changelog.